### PR TITLE
[CI] Propagate workflows from Istio

### PR DESCRIPTION
**Pull request by lolverae**
**Description**

This PR propagates some of the workflows for this adapter, from the Istio adapter, including styling, jobs and version pinning.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by lolverae** [comment]
@leecalcote Let me know how this looks and if I should start making these changes on the rest of adapters just like I've been doing with the meshkit changes.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ updates:
...
   directory: "/"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+    interval: weekly
+  open-pull-requests-limit: 5
   reviewers:
-  - aisuko
-  - Revolyssup
+    - aisuko
```
Comment (leecalcote) [inline .github/dependabot.yml:9]: For as much as we ❤️ aisuko, he's not actively performing this duty any longer and can be removed from this list.

File: .github/workflows/error-ref-publisher.yaml [hunk 2]
```diff
@@ -7,46 +7,45 @@ on:
...
+        # token here with write access to meshery-istio repo
         with:
           token: ${{ secrets.GH_ACCESS_TOKEN }}
-          ref: "master"
+          ref: 'master'
 
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.16'
```
Comment (leecalcote) [inline .github/workflows/error-ref-publisher.yaml:24]: @Revolyssup @acald-creator Yikes!

File: .github/workflows/multi-platform.yml [hunk 3]
```diff
@@ -62,7 +62,7 @@ jobs:
           else
             echo RELEASE_CHANNEL=edge >> $GITHUB_ENV
           fi
-          LATEST_VERSION=$(git ls-remote --sort='v:refname' --tags | tail -1 | cut -f2 | sed 's/refs\/tags\///g') >> $GITHUB_ENV
+           LATEST_VERSION=$(git ls-remote --sort='v:refname' --tags | tail -1 | cut -f2 | sed 's/refs\/tags\///g') >> $GITHUB_ENV
```
Comment (leecalcote) [inline .github/workflows/multi-platform.yml:65]: I wonder if this whitespace is desired. Hmm.

File: .github/workflows/slack.yml [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
       run: |
-        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/meshery/meshery-linkerd' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/${{github.repository}}' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred Meshery Adapter for Linkerd! (https://github.com/meshery/meshery-linkerd/stargazers) Total ⭐️: ${{env.STARS}}\"}'
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred ${{github.repository}}! (https://github.com/${{github.repository}}/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```
Comment (leecalcote) [inline .github/workflows/slack.yml:20]: Gooooood.

**Review by leecalcote — APPROVED** [review]
Only one whitespace looks out of place with the rest looking good to me. 👍

**Comment by leecalcote** [comment]
> @leecalcote Let me know how this looks and if I should start making these changes on the rest of adapters just like I've been doing with the meshkit changes.

These changes look on the money to me, @lolverae. 💯 - 1 (for the unnecessary whitespace), so a 99. :)

**Pull request merged**